nengo/builder/builder.py

```diff
@@ -26,6 +26,7 @@ def __init__(self, dt=0.001, label=None, decoder_cache=NoDecoderCache()):
         self.operators = []
         self.params = {}
         self.seeds = {}
+        self.seeded = {}
```

Comment:
Should we have a brief description here? It took me a bit to figure out what this is actually doing. A few of the others could probably use descriptions, too (of course they weren't part of this PR, so they're not necessarily your job).